Makefile

```diff
@@ -57,6 +57,12 @@ bootstrap:
 		LIB=$$LIB make check-lib ; \
     done
 
+bootstrap-cdk:
+	aws cloudformation describe-stacks \
+		--stack-name CDKToolkit \
```

Comment:
Why is this required and where is this user?

Reply:
This is checking if target account was bootstrapped with CDK (`cdk bootstrap`)